tests/jenkins/lib-testers/TestDownloadFromS3LibTester.groovy

```diff
@@ -0,0 +1,46 @@
+import static org.hamcrest.CoreMatchers.notNullValue
+import static org.hamcrest.MatcherAssert.assertThat
+
+class TestDownloadFromS3LibTester extends LibFunctionTester {
+
+    private String sourcePath
+    private String bucket
+    private String path
+    private boolean force
+
+    public TestDownloadFromS3LibTester(sourcePath, bucket, path, force){
+        this.sourcePath = sourcePath
+        this.bucket = bucket
+        this.path = path
+        this.force = force
+    }
+
+    void parameterInvariantsAssertions(call){
+        assertThat(call.args.sourcePath.first(), notNullValue())
+        assertThat(call.args.bucket.first(), notNullValue())
+        assertThat(call.args.path.first(), notNullValue())
+        assertThat(call.args.force.first(), notNullValue())
+    }
+
+    boolean expectedParametersMatcher(call) {
+        return call.args.sourcePath.first().toString().equals(this.sourcePath)
+                && call.args.bucket.first().toString().equals(this.bucket)
+                && call.args.path.first().toString().equals(this.path)
+                && call.args.force.first() == (this.force)
```

Comment:
```suggestion
                && call.args.force.first().toString().equals(this.force)
```

Reply:
This is boolean why we check if the string is equal()? Like I define on line 9.

Reply:
I think everything inside `call.args.force.first()` is a string. `this.force` is boolean.

Reply:
If that's the case, we will need something like this 
`call.args.force.first().toString().equals(this.force.toString())` to compare the actual context of string for argument `force`